music_assistant/server/providers/opensubsonic/sonic_provider.py

```diff
@@ -100,13 +100,15 @@ async def handle_async_init(self) -> None:
             success = await self._run_async(self._conn.ping)
             if not success:
                 msg = (
-                    f"Failed to connect to {self.config.get_value(CONF_BASE_URL)}, "
+                    "Failed to connect to "
+                    f"{self.config.get_value(CONF_BASE_URL)}, "
                     "check your settings."
                 )
                 raise LoginFailed(msg)
```

Comment:
why not raise AuthError or CredentialError here and catch it in one sweep below ?

Reply:
It felt silly to raise the exception to be caught two lines later and repackaged into the LoginFailed anways, but will do